queries.sql

```diff
@@ -5,4 +5,27 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE (weight_kg >= 10.4 AND weight_kg <= 17.3);
\ No newline at end of file
+SELECT * FROM animals WHERE (weight_kg >= 10.4 AND weight_kg <= 17.3);
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] @sam2if Great work adding a query to add data on species column, however, you forgot to add a query `SELECT species from animals; ` before and after COMMIT to verify the changes were made.